app/src/main/java/xyz/whereuat/whereuat/PendingRequestsActivity.java

```diff
@@ -1,48 +1,55 @@
 package xyz.whereuat.whereuat;
 
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
 import android.database.Cursor;
-import android.database.MatrixCursor;
 import android.os.Bundle;
 import android.support.v4.app.LoaderManager;
 import android.support.v4.content.CursorLoader;
 import android.support.v4.content.Loader;
+import android.support.v4.content.LocalBroadcastManager;
 import android.widget.ListView;
 
 import xyz.whereuat.whereuat.db.entry.PendingRequestEntry;
 import xyz.whereuat.whereuat.ui.adapters.PendingRequestAdapter;
+import xyz.whereuat.whereuat.utils.PendingRequestUtils;
 
 /**
  * This class manages the activity for displaying and interacting with the Pending Requests page.
  */
 public class PendingRequestsActivity extends DrawerActivity implements
         LoaderManager.LoaderCallbacks<Cursor> {
-    PendingRequestAdapter mAdapter;
-    MatrixCursor mCursor;
+    private PendingRequestAdapter mAdapter;
+    private ReloadPendingRequestsReceiver mReloadPendingReqsReceiver;
+    private IntentFilter mReloadPendingReqsFilter;
 
     @Override
     public void onCreate(Bundle saved_instance_state) {
         super.onCreate(saved_instance_state);
         setContentView(R.layout.activity_pending_reqs);
```

Comment:
Toolbar for the pending requests activity should say "Pending Requests" instead of "whereu@"